modules/csg/csg_shape.h

```diff
@@ -169,6 +172,14 @@ class CSGShape3D : public GeometryInstance3D {
 	Ref<ArrayMesh> bake_static_mesh();
 	Ref<ConcavePolygonShape3D> bake_collision_shape();
 
+private:
+	static Callable _navmesh_source_geometry_parsing_callback;
+	static RID _navmesh_source_geometry_parser;
+
+public:
+	static void _navmesh_parse_init();
+	static void _navmesh_parse_source_geometry(const Ref<NavigationMesh> &p_navigation_mesh, Ref<NavigationMeshSourceGeometryData3D> p_source_geometry_data, Node *p_node);
```

Comment:
the types are self-explanatory here, so consider removing those `p_(...)` names

Reply:
Yes they are but that is for consistency as the same naming is used everywhere including the docs.

Reply:
ok